internal/git/gitignore_test.go

```diff
@@ -0,0 +1,130 @@
+package git
+
+import (
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/windmilleng/tilt/internal/testutils"
+)
+
+func TestGitIgnoreTester_Simple(t *testing.T) {
+	tf := newTestFixture(t, ".*.swp")
+	defer tf.TearDown()
+
+	g, err := NewGitIgnoreTester(testutils.CtxForTest(), tf.repoRoots[0].Path())
+	if err != nil {
+		t.Error(err)
+		return
+	}
+
+	ret, err := g.IsIgnored(tf.repoRoots[0].JoinPath("a", "b", ".foo.swp"), false)
+	assert.NoError(t, err)
```

Comment:
not a big deal but i prefer:
```
if assert.NoError(t, err) {
    assert.True(t, ret)
}
```

b/c I only care about the result of the second assert if there's no error -- reporting the second when there IS an error is imo extra noise.

Reply:
yep, thanks